trussworks/react-uswds
# feat: implement ProcessListHeading subcomponent

# Summary

This PR implements the `ProcessListHeading` subcomponent so that users have an out of the box heading component and do not have to manually apply the USWDS heading className.

## Related Issues or PRs

closes #1134 

As the MVP ProcessList PR https://github.com/trussworks/react-uswds/pull/1107 has not yet been merged, I made that branch the target for this PR.

## How To Test

Check out this branch and run `yarn test` to execute tests and `yarn storybook` to see the component in action.

Alternatively, you can scroll to the bottom of this page and click "Show environments" on the right above the comment input box to access this component in Storybook. 

### Screenshots (optional)

Nothing is changed visually in this PR from the MVP ProcessList component PR https://github.com/trussworks/react-uswds/pull/1107
